src/shared/discovery.js

```diff
@@ -115,7 +115,7 @@ export default class Discovery {
     // window and send messages to them.
     const queue = [this.target.top];
     while (queue.length > 0) {
-      const parent = queue.shift();
+      const parent = /** @type {Window} */ (queue.shift());
```

Comment:
is this casting?

Reply:
Yes, exactly. The general syntax is `/** @type {<Type>} */ (<JS expression>)`. The braces around the expression are important.